README.md

```diff
@@ -402,6 +402,7 @@ mechanisms to read column values:
 
 ```go
 pages := column.Pages()
+defer pages.Close()
```

Comment:
This can error, so maybe something like this?

```suggestion
defer checkErr(pages.Close())
```

We don't need to define what `checkErr` does for the docs, but it maybe helpful to remind folks to check `io.Closer` errors.

Reply:
Yep